### ISSUE TITLE
Noisy result on custom object dataset

### ISSUE THREAD

**Issue by Yzhbuaa** [doubleZ0108/GeoMVSNet#16]
Hello, thanks for your great work. I tried to run on my own datset but the result turns out to be noisy:

<img width="1062" alt="image" src="https://github.com/doubleZ0108/GeoMVSNet/assets/44798666/f6490f51-ef04-474a-b25c-cfa8953908db">

Dataset: The dataset is captured by Iphone with image resolution 1080(width) x 1920(height). I captured 61 images for depth estimation. Here is one of the image(I use mosaic and crop for privity):

![image](https://github.com/doubleZ0108/GeoMVSNet/assets/44798666/c827d9a1-dd67-4dc6-b4d2-630e3f0e4476)

Image Poses: I used colmap for pose estimation, and the result seems to be pretty good. Here is the visualization result of the camera poses:

<img width="463" alt="image" src="https://github.com/doubleZ0108/GeoMVSNet/assets/44798666/e435e255-c088-433a-87ff-6b86dd29a17f">

I used the pretrained model which has been trained on dtu dataset provided in the repo. 

Could you provide some instructions on how to remove those noise and make the point cloud clearer? Thank you!

**Comment by doubleZ0108** [comment]
Can you show me some visualization of the depth estimation?

**Comment by doubleZ0108** [comment]
Based on existing observations, you may need to consider the following issues:
1. The effect of depth maps.
2. The parameters of the depth map when fusing point clouds.
3. Point cloud filtering as post-processing, such as statistical filtering, etc.

**Comment by doubleZ0108** [comment]
Another important issue is to control the fixed exposure when using a mobile phone to collect data. For iPhone, it is recommended to download a professional app to control the camera parameters when collecting images, which is crucial for accurate camera parameter estimation and network feature extraction.

**Comment by Yzhbuaa** [comment]
Here is the visualization of the depth estimation: 
![image](https://github.com/doubleZ0108/GeoMVSNet/assets/44798666/14a97803-e97d-42b7-bbf4-83a437464257)

**Comment by Yzhbuaa** [comment]
Could you show me some tips on how to tune the depth map fusion parameters for a better point cloud result? Thank you!

**Comment by doubleZ0108** [comment]
The general effect of the depth map is OK, but it is obviously not accurate enough. It may be that your scene is quite different from the DTU.

For depth map fusion, you can refer to the hyper-parameter in fusion scripts, tune them, and watch the visual mask in the output folder. Generally speaking, the filtered mask can accurately reflect the effect of filtering fusion.
